<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Evangelos-Marios Nikolados</title>
    <link rel="stylesheet" href="main.css">
</head>
<body>
    <div class="page-title">
        <h1>Evangelos-Marios Nikolados</h1>
        <h2>BIOMEDICAL DATA SCIENTIST</h2>
    </div>

    <div class="container small">
        <p>Hi, I’m Evangelos-Marios Nikolados. I hold a PhD in Quantitative Biology, Biochemistry, and Biotechnology from the University of Edinburgh.<grok-card data-id="375292" data-type="citation_card"></grok-card> My research focused on next-generation computational optimization of protein expression systems. I am currently Head of AI at Myria Biosciences.<grok-card data-id="c39640" data-type="citation_card"></grok-card></p>
        
        <p>I previously worked as a Research Assistant at the University of Edinburgh and contributed to Web 3.0 Decentralized Autonomous Organizations (DAOs). I graduated from Harvard University Extension School with a Bachelor of Liberal Arts in Biology.</p>
        
        <p>I enjoy solving challenging problems at the intersection of biology, artificial intelligence, and mathematics. In my free time, I participate in the Greek Scientists Society and organize/sponsor the Global Open Genetic Engineering Competition (GOGEC).</p>
    </div>

    <div class="socials">
        <a href="https://uk.linkedin.com/in/emnikolados"><img src="linkedin-icon.png" alt="LinkedIn"></a>
        <a href="https://scholar.google.com/citations?user=H8fc2JgAAAAJ&hl=en"><img src="google-scholar-icon.png" alt="Google Scholar"></a>
        <a href="https://orcid.org/0000-0003-2303-0889"><img src="orcid-icon.png" alt="ORCID"></a>
    </div>

    <h2>PROGRAMMING SKILLS</h2>

    <div class="skills">
        <div class="left-skills">
            <h3 class="title">Languages</h3>
            <ul>
                <li class="skill">Python (advanced)</li>
                <li class="skill">Matlab (advanced)</li>
                <li class="skill">Julia (intermediate)</li>
                <li class="skill">C (intermediate)</li>
            </ul>
        </div>
    </div>

    <h2>EDUCATION</h2>

    <div class="resume">
        <p><strong>Oct 2019 — 2024</strong></p>
        <p>PhD in Quantitative Biology, Biochemistry, and Biotechnology</p>
        <p>University of Edinburgh</p>
        <p>Research Project: "Next-generation computational optimization of protein expression systems"</p>
        <p>Supervisors: Dr Diego Oyarzún, Prof Meriem El Karoui</p>

        <p><strong>Nov 2018 — March 2019</strong></p>
        <p>Applied Data Science</p>
        <p>Columbia University</p>
        <p>Hands-on course by Columbia Engineering Executive Education and EMERITUS Institute of Management.</p>

        <p><strong>Sep 2017 — Sep 2018</strong></p>
        <p>Master of Research in Systems and Synthetic Biology (Distinction)</p>
        <p>Imperial College London</p>
        <p>Research Project: "Growth defects and loss-of-function in synthetic gene circuits"</p>
        <p>Supervisors: Dr Diego Oyarzún, Dr John Heap</p>

        <p><strong>Jan 2015 — May 2017</strong></p>
        <p>Bachelor of Liberal Arts, Biology</p>
        <p>Extension School, Harvard University - Cambridge, MA</p>
        <p>GPA: 3.87.</p>
        <p>Undergraduate Thesis: "Intrusion Prevention & Detection Models for DNA Data Storage Systems"</p>
        <p>Supervisor: Edward G. Freedman</p>

        <p><strong>Oct 2011 — Oct 2013</strong></p>
        <p>Bachelor of Science in Molecular Biology & Genetics</p>
        <p>Democritus University of Thrace - Alexandroupolis, Greece</p>
        <p>Transferred to Harvard University, Extension School, after 4 semesters of studies.</p>
    </div>

    <h2>WORK EXPERIENCE</h2>

    <div class="resume">
        <p><strong>2024 — Present</strong></p>
        <p>Head of AI</p>
        <p>Myria Biosciences</p>

        <p><strong>Jun 2023 — Nov 2023</strong></p>
        <p>Research Assistant</p>
        <p>Weisse Group, University of Edinburgh</p>
        <p>- Provided domain expertise to the Research Software Engineer.</p>
        <p>- Implemented mechanistic models of bacterial growth in the Julia language.</p>
        <p>- Inspected and provided corrections for the entire code base and documentation.</p>

        <p><strong>Sep 2021 — Present</strong></p>
        <p>Independent Contributor</p>
        <p>Web 3.0</p>
        <p>Contributing in Decentralised Autonomous Organisations (DAOs):</p>
        <p>- GenomesDAO: biotech DAO focused on safe, private, and auditable monetisation of genomic data</p>
        <p>- LabDAO: smart protocols for the exchange of standardised R&D services and scientific protocols and data</p>
        <p>- psyDAO: focusing on funding or research at the intersection of psychedelics and mental health</p>

        <p><strong>Sep 2020 — May 2023</strong></p>
        <p>Project Supervisor</p>
        <p>University of Edinburgh</p>
        <p>- Provided guidance for undergraduate students from the School of Informatics (data preparation & dimensionality reduction, encoding methods, and visualisation) on projects at the intersection of Machine Learning and Biology.</p>
        <p>- Assisted 1st year PhD students from the School of Biological Sciences on Machine Learning pipelines for image analysis and classification of neural stem cells.</p>

        <p><strong>Sep 2016 — Oct 2017</strong></p>
        <p>Tutor</p>
        <p>Quickhelp, Inc. - Boston, MA</p>
        <p>- Taught students study skills, note-taking skills, and test-taking strategies.</p>
        <p>- Provided private instruction online to assist students improve academic performance and prepare for tests.</p>

        <p><strong>Feb 2012 — Jun 2013</strong></p>
        <p>Teacher Assistant</p>
        <p>Democritus University of Thrace, Department of Molecular Biology & Genetics - Alexandroupolis, Greece</p>
        <p>- Tutored students for the courses of Computational Biology and Structural Biology.</p>
        <p>- Instructed and monitored students in the use of equipment to prevent damage.</p>

        <p><strong>Oct 2011 — Mar 2013</strong></p>
        <p>Class President</p>
        <p>Democritus University of Thrace, Department of Molecular Biology & Genetics - Alexandroupolis, Greece</p>
        <p>- Directed and organised educational trips to European Molecular Biology Laboratories (EMBL) and German Cancer Research Center (DKFZ), in Heidelberg, Germany, to expose students to different laboratory environments.</p>
        <p>- Organised four fundraising events for the class trip to Germany. The amount raised (€6000) was split among twenty-two students with financial difficulties.</p>
    </div>

    <h2>PUBLICATIONS</h2>

    <div class="projects">
        <h3>Peer-reviewed</h3>
        <p>E-M Nikolados, and D. A. Oyarzún. 'Deep learning for optimization of protein expression'. Current Opinion in Biotechnology, 2023.</p>
        <p>E-M Nikolados, A Wongprommoon, O. M. Aodha, G. Cambray, and D. A. Oyarzún. 'Accuracy and data efficiency in deep learning models of protein expression'. Nature Communications, 2022.</p>
        <p>E-M. Nikolados, A. Y. Weiße, F. Ceroni, and D. A. Oyarzún. ‘Growth defects and loss-of-function in synthetic gene circuits.’ ACS Synthetic Biology, 2019.</p>

        <h3>Book chapters</h3>
        <p>E-M Nikolados, A. Y. Weiße, D. A. Oyarzún. ‘Prediction of cellular burden with host-circuit models’. In: Menolascina F. (eds) Synthetic Gene Circuits, Methods in Molecular Biology, vol 2229, 2021.</p>
    </div>

    <h2>PRESENTATIONS</h2>

    <div class="resume">
        <p>E-M Nikolados, A Wongprommoon, O. M. Aodha, G. Cambray, and D. A. Oyarzún. 'Deep learning models of protein expression'. Talk for the Darwin Trust Symposium, Edinburgh, 2023.</p>
        <p>E-M Nikolados, A Wongprommoon, O. M. Aodha, G. Cambray, and D. A. Oyarzún. 'Deep learning models of protein expression'. Talk for the Turing Workshop on AI, Engineering Biology & Beyond, Edinburgh, 2023.</p>
        <p>E-M Nikolados, A Wongprommoon, O. M. Aodha, G. Cambray, and D. A. Oyarzún. 'Deep learning models of protein expression'. Flash talk and poster presented at Microbes22: Emerging Applications of Microbes, Leuben, Belgium, 2022.</p>
        <p>E-M Nikolados, A Wongprommoon, O. M. Aodha, G. Cambray, and D. A. Oyarzún. 'Deep learning models of protein expression'. Talk for International Conference on Systems Biology (ICSB) Satellite Event: Synthetic Biology in the Age of Machine Learning in Berlin, Germany, 2022.</p>
        <p>E-M Nikolados, A Wongprommoon, O. M. Aodha, G. Cambray, and D. A. Oyarzún. 'Deep learning models of protein expression'. Poster presented at MOMI22: The World of Industrial Mathematics, INRIA Sophia Antipolis - Mediterranee Research Centre, 2022.</p>
        <p>E-M Nikolados, A Wongprommoon, O. M. Aodha, G. Cambray, and D. A. Oyarzún. 'Deep learning models of protein expression'. Poster presented at EMBO workshop on Codon Usage in Edinburgh, 2022.</p>
        <p>E-M Nikolados, and D. A. Oyarzún, ‘Deep learning for protein expression systems.’ Talk for the Syntetic Biology UK conference in Nottingham, 2021.</p>
        <p>E-M Nikolados, A. Y. Weiße, F. Ceroni, and D. A. Oyarzún, ‘Growth defects and loss-of-function in synthetic gene circuits.’ Poster presented at Workshop on Control Engineering and Synthetic Biology, Worcester College, University of Oxford, 2019.</p>
        <p>E-M Nikolados, D. A. Oyarzún, ‘Growth defects and loss-of-function in synthetic gene circuits.’ Poster presented at Synthetic Biology UK, Bristol, 2018.</p>
    </div>

    <h2>AWARDS & ACHIEVEMENTS</h2>

    <div class="resume">
        <p><strong>University of Edinburgh</strong></p>
        <p>Best poster award at MOMI22 conference (€300).</p>
        <p>DDE Venture Builder Incubator grant (£2000) for KOIOS, a cause & effect discovery NLP pipeline for advanced materials.</p>
        <p>Principal's Career Development Scholarship (focus: Entrepreneurship) jointly awarded by the School of Informatics and School of Biological Sciences.</p>
        <p>Darwin Trust PhD Studentship in Biology and Biomedical Sciences.</p>

        <p><strong>Imperial College London</strong></p>
        <p>Centenary Prize for MSc in Systems & Synthetic Biology.</p>
        <p>Student Member Bursary (£200) by the Biochemical Society, to attend the SBUK 2018.</p>

        <p><strong>Harvard University</strong></p>
        <p>ASL Phi Beta Honor Society</p>
        <p>Dean's List Academic Achievement Award</p>
        <p>Dean's List Honoree, 2015-2016.</p>
        <p>Sponsorship by Konstantinos Naslas, owner of Chronos Shipping Co.LTD (€15000).</p>
        <p>Scholarship for Academic Excellence (2015-2016) by the Union of Masters and Mates of the Greek Merchant Marine (€2000).</p>
        <p>Dean's List Honoree, 2014-2015.</p>

        <p><strong>Democritus University of Thrace</strong></p>
        <p>Scholarship for Academic Excellence (2011-2012) by the State Scholarship Foundation (€1700).</p>
        <p>Scholarship for achieving the best scores in the entry exams to Higher Education Institutions (Jan 18, 2012) by the Union of Masters and Mates of the Greek Merchant Marine (€800).</p>
        <p>Scholarship for achieving the best scores in the entry exams to Higher Education Institutions (Dec 21, 2011) by Eurobank (€1000).</p>
    </div>

    <h2>ADDITIONAL INFO</h2>

    <div class="resume">
        <p><strong>Languages:</strong> English (advanced), Greek (native)</p>
    </div>

    <h2>NOTEBOOK</h2>

    <div class="notebook">
        <p class="title"><a href="blog/medium-article.html">What did Odysseus reply to the Cyclops?</a></p>
        <p class="description">A Medium article on personal insights and coding (cartoonized face with Python)...</p>
    </div>

    <div class="credit">
        <p>built by Evangelos-Marios Nikolados</p>
    </div>
</body>
</html>